pom.xml

```diff
@@ -8,13 +8,13 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-dataflow-parent</artifactId>
-		<version>1.3.0.M2</version>
+		<version>1.3.0.BUILD-SNAPSHOT</version>
 		<relativePath />
 	</parent>
 
 	<properties>
-		<spring-cloud-dataflow.version>1.3.0.M2</spring-cloud-dataflow.version>
-		<spring-cloud-deployer-cloudfoundry.version>1.3.0.M1</spring-cloud-deployer-cloudfoundry.version>
+		<spring-cloud-dataflow.version>1.3.0.BUILD-SNAPSHOT</spring-cloud-dataflow.version>
+		<spring-cloud-deployer-cloudfoundry.version>1.3.0.BUILD-SNAPSHOT</spring-cloud-deployer-cloudfoundry.version>
```

Comment:
I may have misunderstood.  But isn't the  deployer to  be brought to .M2? 
i.e. https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/issues/359

Reply:
Also after discussion both dependencies will be BUILD-SNAPSHOT